lib/services/authority.go

```diff
@@ -710,17 +717,20 @@ func (r *Rotation) CheckAndSetDefaults(clock clockwork.Clock) error {
 // even time periods between rotation phases.
 func GenerateSchedule(clock clockwork.Clock, gracePeriod time.Duration) (*RotationSchedule, error) {
 	if gracePeriod <= 0 {
-		return nil, trace.BadParameter("bad grace period %q, provide value >= 0", gracePeriod)
+		return nil, trace.BadParameter("invalid grace period %q, provide value > 0", gracePeriod)
 	}
 	return &RotationSchedule{
-		UpdateServers: clock.Now().UTC().Add(gracePeriod / 2).UTC(),
+		UpdateClients: clock.Now().UTC().Add(gracePeriod / 3).UTC(),
+		UpdateServers: clock.Now().UTC().Add((gracePeriod * 2) / 3).UTC(),
```

Comment:
Technically, extra () are redundant.

Reply:
yeah, I just wanted it to be very easy to read.